pkg/mount/mountutil.go

```diff
@@ -97,10 +99,14 @@ func (m DiskMountUtil) getDeviceMountAttr(fn getMountData) (DeviceMountAttr, err
 		if !strings.HasPrefix(line, "/dev") {
 			continue
 		}
-		if mountAttr, ok := fn(line); ok {
-			return mountAttr, nil
+		if lineMountAttr, ok := fn(line); ok {
```

Comment:
Here `fn` can either be either [getPartitionName](https://github.com/openebs/node-disk-manager/blob/master/pkg/mount/mountutil.go#L269) or [getMountName](https://github.com/openebs/node-disk-manager/blob/master/pkg/mount/mountutil.go#L288).  Can you test the new approach works in both cases and does not break the existing implementation